src/utils/viewer/viewer.cpp

```diff
@@ -184,10 +164,6 @@ Viewer::Viewer(const Arguments& arguments)
       .setHelp("debug-bullet", "render Bullet physics debug wireframes")
       .addOption("physics-config", ESP_DEFAULT_PHYS_SCENE_CONFIG_REL_PATH)
       .setHelp("physics-config", "physics scene config file")
-      .addBooleanOption("disable-navmesh")
```

Comment:
Don't we want to keep this option?

Reply:
Thanks for bringing this up. Realized several args were not handled.